### PR TITLE
Use CMake 3.19 for RMM when building cuDF jar [skip ci]

### PR DESCRIPTION
**Pull request by GaryShen2008**
Update Dockerfile to install CMake 3.19 as RMM required 3.18+
Add cmake into PATH in build script
Change PTDS default ON

### CODE REVIEW

**Comment by GPUtester** [comment]
Please update the changelog in order to start CI tests.

View the gpuCI docs [here](https://docs.rapids.ai/gpuci#open-pr-or-update-pr).

File: java/ci/Dockerfile.centos7 [hunk 1]
```diff
@@ -36,3 +36,6 @@ RUN cd /rapids/ && wget https://dl.bintray.com/boostorg/release/1.72.0/source/bo
    tar zxf boost_1_72_0.tar.gz && \
    cd boost_1_72_0 && \
    scl enable devtoolset-7 "./bootstrap.sh --prefix=/usr && ./b2 install --with-filesystem threading=multi link=static cxxflags=-fPIC; exit 0"
+
+RUN cd /usr/local/ && wget --quiet https://github.com/Kitware/CMake/releases/download/v3.19.0/cmake-3.19.0-Linux-x86_64.tar.gz && \
+   tar zxf cmake-3.19.0-Linux-x86_64.tar.gz
```
Comment (jlowe) [inline java/ci/Dockerfile.centos7:41]: The install of cmake3 (and link to /usr/bin) should be removed.

Comment (GaryShen2008) [reply java/ci/Dockerfile.centos7:40]: Good catch. Removed.

**Comment by GaryShen2008** [comment]
I have verified the build locally. It's ready to review and merge.

**Review by jlowe — APPROVED** [review]

**Pull request merged**